### ISSUE TITLE
Error in running python mDLAM.py

### ISSUE THREAD

**Issue by Zeinab-E** [xianggebenben/mDLAM#1]
**Hi,
I tried to run mDLAM.py on Windows 10, but I have got the following error:**

 Traceback (most recent call last):
  File "mDLAM.py", line 13, in <module>
    from input_data import cora, pubmed, citeseer,coauthor_cs
  File "/content/mDLAM/input_data.py", line 1, in <module>
    from torch_geometric.datasets import Planetoid, Amazon, Coauthor
  File "/usr/local/lib/python3.7/dist-packages/torch_geometric/__init__.py", line 4, in <module>
    import torch_geometric.data
  File "/usr/local/lib/python3.7/dist-packages/torch_geometric/data/__init__.py", line 1, in <module>
    from .data import Data
  File "/usr/local/lib/python3.7/dist-packages/torch_geometric/data/data.py", line 9, in <module>
    from torch_sparse import SparseTensor
  File "/usr/local/lib/python3.7/dist-packages/torch_sparse/__init__.py", line 41, in <module>
    from .tensor import SparseTensor  # noqa
  File "/usr/local/lib/python3.7/dist-packages/torch_sparse/tensor.py", line 13, in <module>
    class SparseTensor(object):
  File "/usr/local/lib/python3.7/dist-packages/torch/jit/_script.py", line 974, in script
    _compile_and_register_class(obj, _rcb, qualified_name)
  File "/usr/local/lib/python3.7/dist-packages/torch/jit/_script.py", line 67, in _compile_and_register_class
    torch._C._jit_script_class_compile(qualified_name, ast, defaults, rcb)
**RuntimeError: 
Tried to access nonexistent attribute or method 'crow_indices' of type 'Tensor'.:**
  File "/usr/local/lib/python3.7/dist-packages/torch_sparse/tensor.py", line 109
    def from_torch_sparse_csr_tensor(self, mat: torch.Tensor,
                                     has_value: bool = True):
        rowptr = mat.crow_indices()
                 ~~~~~~~~~~~~~~~~ <--- HERE
        col = mat.col_indices()
    

**As you can see, I got this error: "RuntimeError:  Tried to access nonexistent attribute or method 'crow_indices' of type 'Tensor'.:"
I also installed torch==1.8.1, torch_geometric, torch_sparse, torch_scatter.
I would greatly appreciate it if you could help me to tackle this problem.**

**Comment by VistaLee** [comment]
Hi,
According to the following issue, you can try to reinstall torch-sparse (version 0.6.12). Hope this helps!

Issue:
https://github.com/rusty1s/pytorch_sparse/issues/207

**Comment by Zeinab-E** [comment]
Thank you so much. My problem is solved by reinstalling torch-sparse (version 0.6.12). Thanks again!
